### PR TITLE
Add UNICOPE space

### PR DESCRIPTION
**Pull request by copejak**
Add space for UNICOPE  

Etherscan: https://etherscan.io/token/0xc5074d5964de8b8157bf6d4f4846df9bc415fddf
CoinGecko: pending
Website: https://www.unicope.app/
Twitter: https://twitter.com/CopeJak

### CODE REVIEW

File: spaces/unicope/skin.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  --heading-color: white;
+  --border-color: rgba(255,255,255,0.2);
+  --header-bg: #F741F0;
+  --block-bg: transparent;
+
+  #topnav {
+    border-bottom: 4px solid black !important;
+    border-image-slice: 1;
+    border-image-source: linear-gradient(to left, #F741F0, #04B7BC);
+  }
```
Comment (e-asia) [inline spaces/unicope/skin.scss:15]: Please use only theme variables. No extra custom styles are allowed.

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/unicope/skin.scss (hunk 1 above)
Comment (copejak) [reply spaces/unicope/skin.scss]: ok thx, i just deleted it, ill just use the default for now. thx.

File: spaces/unicope/index.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "address": "0xc5074D5964De8B8157BF6D4F4846dF9BC415FDdf",
+  "token": "0xc5074D5964De8B8157BF6D4F4846dF9BC415FDdf",
+  "core": [
+    "0xf8fE0383da886bdd425dd070f6e57Fd21eac1C81"
+  ],
+  "min": 1,
+  "invalid": [],
+      }]
+  ]
+}
```
Comment (e-asia) [inline spaces/unicope/index.json:17]: There is an indentation problem on that file. Please clean it.
```suggestion
  "invalid": []
}
```

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/unicope/index.json (hunk 2 above)
Comment (copejak) [reply spaces/unicope/index.json]: fixed

**Review by e-asia — APPROVED** [review]

**Pull request merged**